metapackages/auto-instrumentations-node/README.md

```diff
@@ -179,6 +179,7 @@ registerInstrumentations({
 - [@opentelemetry/instrumentation-redis](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-redis)
 - [@opentelemetry/instrumentation-restify](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-restify)
 - [@opentelemetry/instrumentation-socket.io](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/instrumentation-socket.io)
```

Comment:
My alphabetical spidey-sense tingled here: I think this list is missing instr-tedious.
Anyway, obviously not something this PR needs worry about.

Reply:
Oof good catch! Yes that's a good follow-up though, will take a look once this merges. At least the instrumentation is in there so that's probably the most important part 😄